kubernetes/corefile-tool/README.md

```diff
@@ -1,74 +1,41 @@
 ## Corefile-tool
 
-Corefile-tool is a simple command line tool which helps you to evaluate and migrate your CoreDNS Corefile Configuration.
-It is based on the [CoreDNS migration tool library](https://github.com/coredns/deployment/tree/master/kubernetes/migration).
+Corefile-tool is a command line tool which helps you to evaluate and migrate your CoreDNS Corefile Configuration.
+It uses the [CoreDNS migration tool library](https://github.com/coredns/deployment/tree/master/kubernetes/migration).
 
 ## Usage
 
 Use the following syntax to run the `corefile-tool` command:
 
-`corefile-tool [command] [flags]`
+```
+Usage:
+    corefile-tool default --corefile <path> [--k8sversion <k8s-ver>]
+    corefile-tool deprecated --from <coredns-ver> --to <coredns-ver> --corefile <path>
+    corefile-tool migrate --from <coredns-ver> --to <coredns-ver> --corefile <path> [--deprecations <true|false>]
```

Comment:
Currently, `--deprecations` is a mandatory flag.

Should this be done as optional and defaults to `false`?

Reply:
Yes